scripts/check-pvm-graph

```diff
@@ -0,0 +1,158 @@
+#!/usr/bin/env fish
+
+if not which neo4j-admin
+	echo "Cannot find neo4j-admin in your current PATH ($PATH)"
+	exit 1
+end
+
+set dehydrated_db_dir $argv[1]
+if [ "$dehydrated_db_dir" = "" ]
+	echo "Usage:  run-neo4j <dehydrated DB directory>"
+	exit 1
+else if not [ -d "$dehydrated_db_dir" ]
+	echo "$dehydrated_db_dir is not a directory"
+	exit 1
+end
+
```

Comment:
Do we want this to take a reference to a file rather than an unzipped directory?

Reply:
Fair point... maybe we ought to support either a directory or a zip file. That shouldn't be too hard... (famous last words)